playground/TestShop/AppHost/Program.cs

```diff
@@ -12,7 +12,10 @@
                             .WithReference(catalogDb)
                             .WithReplicas(2);
 
-var messaging = builder.AddRabbitMQ("messaging").PublishAsContainer();
+var rabbitMqPw = builder.AddParameter("rabbitmqpw", secret: true);
+var messaging = builder.AddRabbitMQ("messaging", password: rabbitMqPw)
+                       .WithDataVolume()
+                       .PublishAsContainer();
```

Comment:
Why is `.PublishAsContainer();` necessary?

Reply:
I didn't add that, it was there already.